types/hapi/index.d.ts

```diff
@@ -898,26 +896,39 @@ export interface ServerStartExtConfigurationObject {
     options?: ServerExtOptions;
```

Comment:
One more as array or method ;)  (Github won't let me comment on the `method: ServerExtFunction` line)